reporting-analysis/src/main/resources/wsdl/envelope.xsd

```diff
@@ -0,0 +1,126 @@
+<?xml version='1.0' encoding='UTF-8' ?>
```

Comment:
do we need these files? I think no
All `wsdl` and `xsd`

Reply:
to remove